### ISSUE TITLE
discriminator loss is negative

### ISSUE THREAD

**Issue by sundyCoder** [sdv-dev/CTGAN#99]
* CTGAN version:
* Python version:
* Operating System:

### Description

Describe what you were trying to get done.
Tell us what happened, what went wrong, and what you expected to happen.

### What I Did

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```

loss_d = -(torch.mean(y_real) - torch.mean(y_fake))

if y_real is bigger than y_fake, the loss_d will be negative value(loss_d < 0), is it correct?

**Comment by fealho** [comment]
Hi @sundyCoder I recommend checking issue #28, which looks similar to what you are asking for.
